solution/0000-0099/0078.Subsets/78. Subsets.js

```diff
@@ -0,0 +1,22 @@
+
+ //* @param {number[]} nums
+ //* @return {number[][]}
+ 
+var subsets = function(nums) {
+    let result=[]
+          
+  const dfs=(i,nums,slate)=>{
+      
+    if(i===nums.length){
+      result.push(slate.slice())
+      return
+    }
+      
+      dfs(i+1,nums,slate)
+      slate.push(nums[i])
+      dfs(i+1,nums,slate)
+      slate.pop()
+  }
+  dfs(0,nums,[])
+  return result
+};
```

Comment:
For the code, we have certain formatting requirements. Prettier configuration files have been added to the project and can be formatted using an IDE.

```suggestion
var subsets = function (nums) {
    let result = [];

    const dfs = (i, nums, slate) => {
        if (i === nums.length) {
            result.push(slate.slice());
            return;
        }

        dfs(i + 1, nums, slate);
        slate.push(nums[i]);
        dfs(i + 1, nums, slate);
        slate.pop();
    };
    dfs(0, nums, []);
    return result;
};
```